exercises/1-User_Mode/challenge_101/test/test_101_challenge.py

```diff
@@ -0,0 +1,131 @@
+"""Define test cases for Challenge 101.
+
+Define test cases for User Mode challenge 101: Signal handlers.
+
+    Typical usage example:
+
+    cd exercises/1-User_Mode/challenge_101
+    python3 -m unittest -v                 # Run all the test cases in verbose mode
+    # -or-
+    python3 -m unittest -v -k boundary_01  # The one test cases that should *always* pass
+    # -or-
+    python3 -m unittest -v -k normal_01    # If you can make this pass, you're doing well
+"""
+
+# Standard Imports
+# Third Party Imports
+from test.challenge_101_test_class import Challenge101Test, SignalResults
+from hobo.subprocess_wrapper import execute_subprocess_cmd
+# Local Imports
+
+
+class Challenge101Normal(Challenge101Test):
+    """Defines normal test cases."""
+
+    def test_normal_01(self):
+        """Start, SIGQUIT, then SIGKILL.
+
+        Test class will implicitly add a SIGKILL.
+        """
+        self.check_signals([SignalResults(3, False)])
+        self.expect_stderr(['Ignoring signal: 3'])
+        self.run_test()
+
+    def test_normal_02(self):
+        """Start, SIGINT, then SIGKILL.
+
+        Explicitly added SIGKILL."""
+        self.check_signals([SignalResults(2, False), SignalResults(9, True)])
+        self.expect_stderr(['Ignoring signal: 2'])
```

Comment:
Change expected output to just be 'Ignoring signal' since printf() is not a viable solution for signal handlers.

Reply:
Updated.